src/core/metallicroughnesseffect.cpp

```diff
@@ -351,15 +351,6 @@ MetallicRoughnessEffect::MetallicRoughnessEffect(Qt3DCore::QNode *parent)
     addTechnique(m_metalRoughES3Technique);
     addTechnique(m_metalRoughES2Technique);
 
-    // Note that even though those parameters are not exposed in the API,
-    // they need to be kept around for now due to a bug in some drivers/GPUs
-    // (at least Intel) which cause issues with unbound textures even if you
-    // don't try to sample from them.
-    // Can probably go away once we generate the shaders and deal in this
-    // case in a better way.
-    addParameter(new QParameter(QStringLiteral("envLight.irradiance"), new QTexture2D));
-    addParameter(new QParameter(QStringLiteral("envLight.specular"), new QTexture2D));
-
```

Comment:
I'm not sure we can remove it now... I mean due to ABI compatibility it could run on any 5.12.x version and only 5.12.2 will carry the fix. I'd be tempted to make that conditional based on 5.13 being detected at build time.

Reply:
done